### PR TITLE
Added configuration in order to publish to the gradle repo. 

### PR DESCRIPTION
**Pull request by jjvilleg**
Pushed to the gradle repo. 

https://plugins.gradle.org/plugin/net.wasdev.wlp.gradle.plugins.Liberty

Please let me know of any wording suggestions you may have. Once everything looks good, I'll finish setting up the jenkins job in order to automatically deploy.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -74,18 +90,32 @@ task sourcesJar(type: Jar) {
...
-    signing {           
+    signing {
         sign configurations.archives
     }
 }
 
+pluginBundle {
+  website = 'https://github.com/WASdev/ci.gradle'
+  vcsUrl = 'https://github.com/WASdev/ci.gradle'
+  description = 'Gradle plugins for managing Liberty profile servers'
```
Comment (cthigh) [inline build.gradle]: Remove "profile"

File: build.gradle [hunk 2]
```diff
@@ -74,18 +90,32 @@ task sourcesJar(type: Jar) {
...
+  tags = ['liberty', 'websphere', 'devops']
+
+  plugins {
+    ciGradle {
+      id = 'net.wasdev.wlp.gradle.plugins.Liberty'
+      displayName = 'ci.Gradle'
```
Comment (cthigh) [inline build.gradle]: Not sure where one would see this name, but wondering if the G should be upper case.  Be consistent.

File: build.gradle (hunk 1 above)
Comment (cthigh) [reply build.gradle]: also "plugin" rather than "plugins"

File: build.gradle [hunk 3]
```diff
@@ -23,10 +35,14 @@ dependencies {
     compile localGroovy()
     compile ('net.wasdev.wlp.ant:wlp-anttasks:1.4-SNAPSHOT')
     compile group: 'commons-io', name: 'commons-io', version: '2.5'
+    compile gradleApi()
+    compile localGroovy()
     provided group: 'com.ibm.websphere.appserver.api', name: 'com.ibm.websphere.appserver.spi.kernel.embeddable', version: '1.0.0'
     testCompile 'junit:junit:4.11'
 }
 
+apply plugin: "com.gradle.plugin-publish"
```
Comment (cthigh) [inline build.gradle]: should this be up with the other plugins?

Comment (jjvilleg) [reply build.gradle]: Moved to the top with the other plugins.

File: build.gradle (hunk 1 above)
Comment (jjvilleg) [reply build.gradle]: Took this from the description on the github page. Went ahead and changed this as well.

File: build.gradle (hunk 2 above)
Comment (jjvilleg) [reply build.gradle]: I changed it in both places. For some reason, I thought the G was capitalized in the github page.

**Pull request merged**